cloudinary/apiCaller.js

```diff
@@ -73,6 +78,9 @@ const sendToCloudinery = (imagesArray, batchSize, dpr, metaData, quality, cb, ro
       cb(parsed, null);
       return;
     }
+    // move lcp
+    const lcpIdx = parsed.imagesTestResults.findIndex((i) => i.tags.includes('lcp'));
+    metaData.lcp = parsed.imagesTestResults.splice(lcpIdx, 1)[0];
```

Comment:
What if no Lcp was found?

Reply:
I  think that will always return an LCP but I think it is more of a product question of what to do if there is no LCP